### ISSUE TITLE
NEW VERSION - looking for testers / contributors

### ISSUE THREAD

**Issue by janpaepke** [johnpolacek/superscrollorama#94]
Hi Guys,

I just finished working on the very first pre-release version of ScrollMagic, a complete rewrite of Superscrollrama.

You can find it here: https://github.com/janpaepke/ScrollMagic

If you are willing and able please test it out and post any issues in the repo's issue list.
There's plenty of new stuff: on the fly modification of all scene parameters, including the trigger, fixed the pin-spacer-expand issue, smooth tweening option, oop handling, debug logging and some more...

Here's a basic how-to-use:
Instead of having one controller and adding tweens or pins with tweens to it ScrollMagic makes a difference between the controller and can have multiple scenes.
A Scene can be a pin, an tween or both.

Basically you add the controller first, like this

```
var controller = new ScrollMagic();
```

And then you can start adding scenes to it.

```
var scene = new ScrollScene({
        duration: 400,
        triggerElement: "#trigger"
    })
    .addTo(controller)
    .setTween(TweenMax.to("#ani", .5, {color: "red" }));
```

When you pull the repo you'll find a test.html with some of my testing code.
Also in the doc folder you'll find extensive (but of course not yet complete) documentation.

Now go and do stuff! :)

regards,
Jan
